src/main.c

```diff
@@ -192,6 +190,60 @@ void kill_childs()
 	debug(D_EXIT, "All threads/childs stopped.");
 }
 
+struct option_def options[] = {
+	// opt description                                                       arg name                     default value
+	{'c', "Load alternate configuration file",                               "config_file",                          CONFIG_DIR "/" CONFIG_FILENAME},
+	{'D', "Disable fork into background",                                    NULL,                                   NULL},
+	{'h', "Display help message",                                            NULL,                                   NULL},
+	{'P', "File to save a pid while running",                                "FILE",                                 NULL},
+	{'p', "Port to listen. Can be from 1 to 65535.",                         "port_number",                          "19999"},
+	{'s', "Path to access host /proc and /sys when running in a container.", "PATH",                                 NULL},
+	{'t', "The frequency in seconds, for data collection. \
+Same as 'update every' config file option.",                                 "seconds",                              "1"},
+	{'l', "The number of entries the netdata daemon will keep in memory \
+for each chart dimension. Same as 'history config file option.",             "lines_to_save",                        NULL},
```

Comment:
I don't think anyone is using this options. We can rename it (anything like history, db size, entries, etc will do), or just remove it. Actually I don't like this options at all. If you set it, the database will be emptied, which is bad. So I am voting for its removal.